Data/HashSet/Base.hs

```diff
@@ -282,6 +283,10 @@ fromMap = HashSet
 keysSet :: HashMap k a -> HashSet k
 keysSet m = fromMap (() <$ m)
 
+-- | /O(n*m)/ Subset of sets.
+subset :: (Eq a, Hashable a) => HashSet a -> HashSet a -> Bool
```

Comment:
The `containers` variant is named [`isSubsetOf`](http://hackage.haskell.org/package/containers-0.6.2.1/docs/Data-Set.html#v:isSubsetOf).